### DISCUSSION TITLE
posts/Deploy-Java-Web-Application-to-heroku/

### DISCUSSION THREAD

**Discussion by giscus** [vegetaz/vegetaz.github.io#9] [General]
# posts/Deploy-Java-Web-Application-to-heroku/

heroku là một dịch vụ máy chủ, đám mây hỗ trợ một số ngôn ngữ lập trình. Trong đó có Java, NodeJS, PHP, …

https://vegetaz.github.io/posts/Deploy-Java-Web-Application-to-heroku/

**Comment by vegetaz** [comment]
Được không?

**Reply by vegetaz** [reply]
À được :v
